### DISCUSSION TITLE
Cannot hide QSelect options dialog header with scoped css in Cordova App

### DISCUSSION THREAD

**Discussion by nulele** [quasarframework/quasar#9471] [CLI - Capacitor / Cordova modes]
Hello,
in my **Quasar Cordova App**, I have a component with a QSelect and I'd like to hide the options dialog header but cannot do it with the _scoped_ attribute.

See the image for html structure
https://postimg.cc/62HCJzh2

NOT WORKS
```
<style scoped>
  div.q-select__dialog >>> label {
    display: none;
  }
</style>
```

WORKS
```
<style>
  div.q-select__dialog label {
    display: none;
  }
</style>
```

Any idea?
Thanks

**Comment by metalsadman** [comment]
I think you should deep select q-select's class first. ie. `v::deep.q-select__dialog label`

**Reply by nulele** [reply]
Thanks but it doesn't work... Maybe I've already tried something like that.

Every time I re-build the app (in dev mode) to be super sure because I know hot reload is not trustable in this situation.
